# Message for hip servo controller

float64 theta_hr
float64 theta_hl